src/jmcomic/jm_config.py

```
@@ -382,11 +404,29 @@ def get_fix_ts_token_tokenparam(cls):
+                try:
+                    sig = inspect.signature(executor)
+                    params_count = len(sig.parameters)
+                except (ValueError, TypeError):
+                    params_count = 2
+
+                if params_count >= 3:
+                    executor(topic, msg, e)
+                else:
+                    import warnings
+                    warnings.warn(
+                        'jmcomic已升级到标准logging，建议将EXECUTOR_LOG重新定义为3个参数以接收异常对象 (topic, msg, e)',
```

Ruff (0.15.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)